aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1026,19 +1026,8 @@ static std::pair<Tensor, Tensor> _not_zero_mask_to_col_row_indices(
     Tensor not_zero_mask,
     ScalarType index_dtype,
     Device index_device) {
-  auto col_indices =
-      at::native::arange(
-          not_zero_mask.size(-1), index_dtype, kStrided, index_device)
```

Comment:
All these were the last 2 indices here... Are we sure the dimensional of the shape is right on higher dim sparse inputs?